src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -9843,7 +9843,7 @@
 				34079851274D3D41007E6071 /* HealthCertificateService+UITestingMethods.swift in Sources */,
 				BA5B7FD626458EF400502087 /* HealthCertificateSimpleTextCellViewModel.swift in Sources */,
 				01B7232F24FE4F080064C0EB /* OptionGroupViewModel.swift in Sources */,
-				ABC506CF274BE30700698A95 /* Locator+ServiceIdentityDocument.swift in Sources */,
+				ABC506CF274BE30700698A95 /* ServiceIdentityDocument.swift in Sources */,
```

Comment:
I think this name is misleading. It looks like its the "ServiceIdentityDocument". But it only contains the extension for the locator. I would propose to keep "Locator+ServiceIdentityDocument.swift". And rename the other files to the same schema.

Reply:
Or maybe other way around ""ServiceIdentityDocument+Locator.swift""